include/benchmark/benchmark.h

```diff
@@ -1090,6 +1090,10 @@ class Benchmark {
 
   virtual void Run(State& state) = 0;
 
+  TimeUnit GetTimeUnit() const;
+
+  static void SetDefaultTimeUnit(TimeUnit unit);
```

Comment:
```suggestion
  static void DefaultUnit(TimeUnit unit);
```
to be better aligned with `Benchmark* Unit(TimeUnit unit)`

Reply:
On the other hand I'm not sure this is the best place for the default, and not around instance / reporter, but let's see what the authors respond.

Reply:
The way these methods are phrased so that you can write benchmarks like this `BENCHMARK(...)->Unit(...)->Arg(...)`
Calling DefaultUnit is not valid in that case. Maybe I should write these as global free functions with a global TimeUnit variable in the .cc file. There are already examples in the code for global functions. Also that would make the SetDefaultTimeUnit function to be out of the internal namespace. Wdyt?

Reply:
644ae69ddda3370d7e9322bdeca71e31914e4d3a